README.md

```diff
@@ -5,6 +5,68 @@ https://flutter.dev/
 Flutter in an open Source SDK for creating high-performance mobile apps for IOS and Android.
 The Flutter makes it easier for you to build user interfaces, while reducing the amount of code required to create and update your app.
 
+## Why use Flutter?
+
+- Flutter can be used to build applications for Android and IOS
```

Comment:
```suggestion
- Flutter can be used to build cross platform native applications (Android, IOS, Desktop and Web) using the same codebase.
```

Reply:
You're totally right after seeing more that you want to pass Flutter not only to Android and IOS but also to other platforms